modules/Campaigns/WizardHome.php

```diff
@@ -213,33 +170,33 @@ function getMarketingId()
     global $app_list_strings;
     global $app_strings;
     global $current_user;
-    
+
```

Comment:
Hi @Dillon-Brown, do you think its safe to remove the commented code

```
    /*    $header_URL = "Location: index.php?module=Campaigns&action=index";
        $GLOBALS['log']->debug("about to post header URL of: $header_URL");
        header($header_URL);
    */
```

Reply:
Yup, pushed!